src/main/java/com/redhat/devtools/intellij/common/CommonConstants.java

```diff
@@ -17,4 +17,5 @@
     public static final String HOME_FOLDER = System.getProperty("user.home");
 
     public static final Key<Project> PROJECT = Key.create("project");
+    public static final Key<Long> LAST_MODIFICATION_STAMP = Key.create("last.modification.stamp");
```

Comment:
After the fact, I realize that this must be namespaced to make sure other plugins may not overrwrite those value. So should be prefixed by com.redhat.devtools.intellij.common